src/commands/infra/stack/new.js

```diff
@@ -39,7 +39,7 @@ FunctionsStackNew.args = [
 
 FunctionsStackNew.flags = Object.assign(
   {},
-  convertYargsOptionsToOclifFlags(cliInfo.options),
+  convertYargsOptionsToOclifFlags(options),
```

Comment:
Hmm, I checked out your branch, uninstalled the twilio-infra plugin then reinstalled. I'm still seeing the error:
```
(node:66188) ReferenceError Plugin: plugin-twilio-infra: cliInfo is not defined
module: @oclif/config@1.17.0
task: toCached
plugin: plugin-twilio-infra
root: /Users/mbermudez/.twilio-cli/node_modules/plugin-twilio-infra
See more details with DEBUG=*
Installing plugin https://github.com/kaiquelupo/plugin-twilio-infra... installed v0.0.1
```

Is this the right installation command? `twilio plugins:install kaiquelupo/plugin-twilio-infra`

Reply:
@bermudezmt are you testing the changes using:

```shell
twilio plugins:link <path_to_your_local_repo>
````

From your log it looks like you are still fetching the repo from github.

Reply:
Yeah you're right. It worked but after running `npm install` first. :) I created a PR  for updating the README. Can you please review? I can't tag you somehow.